prow/config.yaml

```diff
@@ -3068,6 +3070,7 @@ presubmits:
     cluster: security
     context: pull-security-kubernetes-bazel-test
     labels:
+      preset-bazel-scratch-dir: "true"
```

Comment:
why are some of these `true` and some of these `"true"`?

Reply:
the security repo is auto generated

Reply:
most of our hand written stuff is overly quoted, the security repo has fully auto generated config so it gets whatever quoting is left after being shoved through go's yaml and a python lib.

Reply:
/shrug